js/main.js

```diff
@@ -14,51 +12,81 @@ function storageAvailable(type) {
   } catch (e) {
     return e instanceof DOMException && (
       e.code === 22
-      || e.code === 1014
-      || e.name === 'QuotaExceededError'
-      || e.name === 'NS_ERROR_DOM_QUOTA_REACHED')
+        || e.code === 1014
+        || e.name === 'QuotaExceededError'
+        || e.name === 'NS_ERROR_DOM_QUOTA_REACHED')
       && (storage && storage.length !== 0);
   }
 }
 
-function updateLocalStorage() {
-  // Updates the Local Storage
-  if (storageAvailable('localStorage')) {
-    localStorage.setItem('books', JSON.stringify(books));
-  }
-}
+class Book {
+  // Book Class has a title and author
+  title;
 
-const localBooksData = localStorage.getItem('books');
+  author;
 
-// Load initially stored data
-if (localBooksData) {
-  books = JSON.parse(localBooksData);
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
 }
 
-// Books Related Values & Functions
-const booksList = document.getElementById('books-list');
+class Library {
+  // Library class consists of a list of books
+  books = [];
 
-function bookExists(book) {
-  // Check if a book exists
-  for (let i = 0; i < books.length; i += 1) {
-    if (books[i].title === book.title && books[i].author === book.author) {
-      return true;
+  bookExists(book) {
+    // Check if a book exists
+    for (let i = 0; i < this.books.length; i += 1) {
+      if (this.books[i].title === book.title && this.books[i].author === book.author) {
+        return true;
+      }
     }
+    return false;
   }
-  return false;
-}
 
-function removeBook(book) {
-  // Remove a book from the list of lists
-  for (let i = 0; i < books.length; i += 1) {
-    if (books[i].title === book.title && books[i].author === book.author) {
-      books.splice(i, 1);
-      updateLocalStorage();
+  addBook(book) {
+    // Add a new book to the list of books
+    if (!this.bookExists(book)) {
+      // eslint-disable-next-line no-use-before-define
+      displayNewElement(book);
+      this.books.push(book);
```

Comment:
- [OPTIONAL] 
In this part it will be better to use array methods that doesn't mutate the array, like `concat` method instead of `push` 

i.e. : 
```JS
this.books = this.books.concat({ title, author, author });
```

Reply:
![image](https://user-images.githubusercontent.com/73714615/152990789-645a0440-1a63-4522-8358-f24ede0a0d75.png)

So this is what I found on google. I believe that the push method is good here since we are just adding a new element at the end of the array.